devops/production-template/provision.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+# Download Ansible roles
+ansible-galaxy install \
+  --force \
+  --role-file=./requirements.yml \
+  --roles-path=./roles
+
+# Run Ansible playbook
+export ANSIBLE_HOST_KEY_CHECKING=False
```

Comment:
Is it best practice to disable host key verification? Forgive my ansible ignorance -- just wondering what problem this is meant to get around.

Reply:
It removes an interactive prompt for the user to accept, particularly if this is run in CI. Otherwise, The first time a machine is connected to, it will always prompt to accept.

Reply:
That's what I assumed.